assemblies/DataSources/OpenConfig/Skyline.DataMiner.DataSources.OpenConfig.Gnmi.Protocol.xml

```diff
@@ -521,32 +1014,72 @@
         </member>
         <member name="M:Skyline.DataMiner.DataSources.OpenConfig.Gnmi.Protocol.DataMapper.DataMinerConnectorDataMapper.ReadExistingValues">
             <summary>
             Reads out the parameter values and stores them internally.
             </summary>
         </member>
-        <member name="M:Skyline.DataMiner.DataSources.OpenConfig.Gnmi.Protocol.DataMapper.DataMinerConnectorDataMapper.SetLogger(Skyline.DataMiner.DataSources.OpenConfig.Gnmi.Utils.ILogger)">
+        <member name="M:Skyline.DataMiner.DataSources.OpenConfig.Gnmi.Protocol.DataMapper.DataMinerConnectorDataMapper.RemoveMissingRowsForPid(System.Int32)">
+            <summary>
+            Removes all rows from the table that have the deleted status.
+            </summary>
+            <param name="tablePid">Parameter ID of the table for which the rows will need to be removed.</param>
+        </member>
+        <member name="M:Skyline.DataMiner.DataSources.OpenConfig.Gnmi.Protocol.DataMapper.DataMinerConnectorDataMapper.RemoveMissingRowsForPid(System.Int32,System.Collections.Generic.IEnumerable{System.String})">
+            <summary>
+            Removes the rows from the table that are present in the <paramref name="pks"/> on the condition that they also have the deleted status.
+            </summary>
+            <param name="tablePid">Parameter ID of the table for which the rows will need to be removed.</param>
+            <param name="pks">Primary keys to be removed when the state of the row indicates that the row is deleted.</param>
+        </member>
+        <member name="M:Skyline.DataMiner.DataSources.OpenConfig.Gnmi.Protocol.DataMapper.DataMinerConnectorDataMapper.RemoveMissingRowForPid(System.Int32,System.String)">
+            <summary>
+            Removes the row from the table where the pk is equal to <paramref name="pk"/> on the condition that the row also has the deleted status.
+            </summary>
+            <param name="tablePid">Parameter ID of the table for which the rows will need to be removed.</param>
+            <param name="pk">Primary key to be removed when the state of the row indicates that the row is deleted.</param>
+        </member>
+        <member name="M:Skyline.DataMiner.DataSources.OpenConfig.Gnmi.Protocol.DataMapper.DataMinerConnectorDataMapper.SetLogger(Skyline.DataMiner.DataSources.CommunicationGatewayMiddleware.Common.Api.ILogger)">
             <summary>
             Assigns a logger interface.
             </summary>
             <param name="logger">Logger interface</param>
         </member>
         <member name="M:Skyline.DataMiner.DataSources.OpenConfig.Gnmi.Protocol.DataMapper.DataMinerConnectorDataMapper.SuppressCommunication(System.Boolean)">
             <summary>
-            Suppresses the communication, no more values will be filled in or logging will be written out.
+            Suppresses the communication. No more values will be filled in or logging will be written out.
```

Comment:
Do you mean no more values will be filled in and no logging will be written out?